spec/requests/routes_crud_spec.rb

```diff
@@ -109,6 +115,30 @@
       expect(route.backend_id).to eq("a-backend")
     end
 
+    describe "updating the disabled flag" do
+      it "should set the disabled flag to the value given in the request" do
+        put_json "/routes", :route => {:incoming_path => "/foo/bar", :disabled => true}
+
+        expect(response.code.to_i).to eq(200)
+        expect(JSON.parse(response.body)).to include({"disabled" => true})
+
+        route = Route.where(:incoming_path => "/foo/bar").first
+        expect(route.disabled).to eq(true)
+      end
+
+      it "should not change the disabled flag when not specified in the request" do
```

Comment:
Think this also needs a reverse of this test, ie specifying only incoming_path and disabled does not change the other existing fields (backend, route_type).